client/js/views/GameView.vue

```diff
@@ -1354,7 +1354,7 @@ export default {
   min-width: 50%;
   height: 30vh;
   & #player-hand-cards,
-  #player-hand-targeting::v-deep {
+  :deep(#player-hand-targeting) {
```

Comment:
After upgrading to Vue 2.7 this warning was being thrown in the console so I would really appreciate you double checking the styles since I'm not entirely sure how deep works in this regard.